src/twisted/newsfragments/11753.misc

```diff
@@ -0,0 +1 @@
+Improve the type annotation of Deferred.fromFuture.
```

Comment:
```suggestion
twisted.internet.defer.Deferred.fromFuture now has a more precise type annotation.
```

Reply:
Note that ".misc" fragments are usually empty because their content will not be included in the release notes.

Reply:
> Note that ".misc" fragments are usually empty because their content will not be included in the release notes.

What would you suggest: should I just empty out this one, or would this fit better in another section?

Reply:
It's public-facing so I'd say go ahead and move it to another section so the details show up.